website/docs/r/metric_ruleset.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "signalfx"
+page_title: "SignalFx: signalfx_metric_ruleset"
+sidebar_current: "docs-signalfx-resource-metric-ruleset"
+description: |-
+Allows Terraform to create and manage SignalFx Metric Rulesets
+---
+
+# Resource: signalfx_metric_ruleset
+
+Provides an Observability Cloud resource for managing Metric Rulesets
```

Comment:
"Provides a Splunk Observability Cloud resources for managing metric rulesets"

I wouldn't capitalize metric rulesets, but I'm not sure what the official formatting should be?